pkg/clients/subnetwork/subnetwork.go

```diff
@@ -17,21 +17,29 @@ limitations under the License.
 package subnetwork
 
 import (
+	"github.com/google/go-cmp/cmp"
+	"github.com/google/go-cmp/cmp/cmpopts"
 	"google.golang.org/api/compute/v1"
 
-	"github.com/crossplaneio/stack-gcp/apis/compute/v1alpha3"
+	"github.com/crossplaneio/crossplane-runtime/pkg/resource"
+
+	"github.com/crossplaneio/stack-gcp/apis/compute/v1beta1"
+	gcp "github.com/crossplaneio/stack-gcp/pkg/clients"
 )
 
+var equateGCPSubnetworkSecondaryRange = func(i, j *v1beta1.SubnetworkSecondaryRange) bool { return i.RangeName > j.RangeName }
```

Comment:
So it is named this because there is a separate `equateSubnetworkSecondaryRange` func in the test file (which sorts the GCP representation of a `SecondaryRange`, but I think it would be more logical to change the one here to `equateSecondaryRange` and the test one `equateGCPSecondaryRange`.